backend/src/test/java/com/team4/backend/repository/StudentRepositoryTest.java

```diff
@@ -32,8 +35,30 @@
     @BeforeAll
     void init() {
         Flux<Student> users = Flux.just(
-                Student.studentBuilder().email("testing_1@gmail.com").password("password1").build(),
-                Student.studentBuilder().email("testing_2@gmail.com").password("password2").build()
+                Student.studentBuilder()
```

Comment:
Move to StudentMockData

Reply:
is not our convention because it is not a mock data, go check the other repository test

Reply:
👍